exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -254,20 +259,22 @@ func (prwe *prwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 			return err
 		}
 		defer resp.Body.Close()
-
 		// 2xx status code is considered a success
 		// 5xx errors are recoverable and the exporter should retry
 		// Reference for different behavior according to status code:
 		// https://github.com/prometheus/prometheus/pull/2552/files#diff-ae8db9d16d8057358e49d694522e7186
 		if resp.StatusCode >= 200 && resp.StatusCode < 300 {
+			prwe.settings.Logger.Debug(fmt.Sprintf("Successfully sent metrics to %s", prwe.clientSettings.Endpoint))
 			return nil
 		}
 
 		body, err := io.ReadAll(io.LimitReader(resp.Body, 256))
-		rerr := fmt.Errorf("remote write returned HTTP status %v; err = %w: %s", resp.Status, err, body)
+		rerr := fmt.Errorf("remote write returned HTTP status %v; url: %s; err = %w: %s", resp.Status, req.URL, err, body)
```

Comment:
I'm not sure adding the target URL is valuable here as it cannot change during the lifetime of a process and is controlled by configuration.  It could be logged at exporter startup for convenience, but when this error is logged it should be in a context that includes the component identifier which is sufficient to point to the configuration that contains this information.